Dockerfile

```diff
@@ -85,3 +85,7 @@ COPY --from=trivy-scanner-build /workspace/out/trivy-scanner /
 USER 65532:65532
 WORKDIR /var/lib/trivy
 ENTRYPOINT ["/trivy-scanner"]
+
+FROM scratch as non-vulnerable
+COPY . .
```

Comment:
why are we copying?

Reply:
the image needed something there so that it wasn't empty, it throws an error with just the "FROM scratch as non-vulnerable". Is there another way I should build it?

Reply:
can we just add `LABEL`?

Reply:
I think it errors if there's no layers.

Reply:
Maybe in a builder image you can just produce an emtpy file and copy that in here.